samples/apps/forum/tsconfig.json

```diff
@@ -11,6 +11,12 @@
         "preserveConstEnums": true,
         "sourceMap": false
     },
+    "ts-node": {
+        "compilerOptions": {
+            "lib": ["ES2015"],
```

Comment:
Why ES2015?

Reply:
Because everything is green for ES2015: https://node.green/

Reply:
Oh I see, I was confused and though this was to do with QuickJS, nevermind.